cmd/substreams/init.go

```diff
@@ -188,21 +243,23 @@ func runSubstreamsInitE(cmd *cobra.Command, args []string) error {
 				Key:   key,
 				Value: gen,
 			}
-			options = append(options, entry)
+			generatorOptions = append(generatorOptions, entry)
 		}
 
+		fmt.Printf("Testing................")
 		var codegen *pbconvo.DiscoveryResponse_Generator
-		selectField := huh.NewSelect[*pbconvo.DiscoveryResponse_Generator]().
-			Title("Choose the code generator that you want to use to bootstrap your project").
-			Options(options...).
+		selectGeneratorField := huh.NewSelect[*pbconvo.DiscoveryResponse_Generator]().
+			Title("Choose the chain ecosystem that you want to use to bootstrap your project (EVM, Cosmos, Solana, etc)").
```

Comment:
What the difference with `Choose the protocol you inted to work with:`? Isn't this question to choose a specific generator, even now?